tests/integration/olp-cpp-sdk-core/DefaultNetworkTest.cpp

```diff
@@ -134,4 +154,146 @@ TEST(DefaultNetworkTest, Send) {
   }
 }
 
+TEST(DefaultNetworkTest, DefaultBucket) {
+  SCOPED_TRACE(
+      "Network response download/upload are appended to default bucket");
```

Comment:
No need for such big sentences, just use "encrypted" messages, e.g. "Default bucket used"